server/src/test/java/org/elasticsearch/index/mapper/FieldAliasMapperValidationTests.java

```diff
@@ -176,7 +176,9 @@ private static ObjectMapper createObjectMapper(String name) {
     }
 
     private static NestedObjectMapper createNestedObjectMapper(String name) {
-        return new NestedObjectMapper.Builder(name, IndexVersion.current()).build(MapperBuilderContext.root(false, false));
+        return new NestedObjectMapper.Builder(name, IndexVersion.current(), query -> { throw new UnsupportedOperationException(); }).build(
```

Comment:
Nit: consider adding a helper for `query -> { throw new UnsupportedOperationException();` to avoid repetition.